core/src/main/scala/org/apache/spark/scheduler/dynalloc/ExecutorMonitor.scala

```diff
@@ -227,7 +227,7 @@ private[spark] class ExecutorMonitor(
     }
 
     if (updateExecutors) {
-      val activeShuffleIds = shuffleStages.map(_._2).toSeq
+      val activeShuffleIds = shuffleStages.map(_._2)
```

Comment:
Just double checking you're sure these work in 2.13? b/c I added a lot of calls like this that will be redundant in 2.12 but not 2.13. (Not saying this one is necessary, particularly.)

Reply:
Yes, I'm sure this conversion is redundant, the type of `shuffleStages` is `Seq[(Int, Int)]` in Scala 2.13, and I run 
`mvn clean install -Pscala-2.13 -pl core -am` locally, all test passed in core module

```
Run completed in 36 minutes, 54 seconds.
Total number of tests run: 2710
Suites: completed 270, aborted 0
Tests: succeeded 2710, failed 0, canceled 4, ignored 7, pending 0
All tests passed.
```
I will do more more local tests to verify other changes today

Reply:
`mvn clean install -Phadoop-3.2 -Phive-2.3 -Phadoop-cloud -Pmesos -Pyarn -Pkinesis-asl -Phive-thriftserver -Pspark-ganglia-lgpl -Pkubernetes -Phive -Pscala-2.13 -pl graphx`

all test passed

```
Run completed in 2 minutes, 24 seconds.
Total number of tests run: 108
Suites: completed 19, aborted 0
Tests: succeeded 108, failed 0, canceled 0, ignored 0, pending 0
All tests passed.
```

Reply:
`mvn clean install -Phadoop-3.2 -Phive-2.3 -Phadoop-cloud -Pmesos -Pyarn -Pkinesis-asl -Phive-thriftserver -Pspark-ganglia-lgpl -Pkubernetes -Phive -Pscala-2.13 -pl mllib
`
all test passed
```
Run completed in 28 minutes, 39 seconds.
Total number of tests run: 1627
Suites: completed 206, aborted 0
Tests: succeeded 1627, failed 0, canceled 0, ignored 7, pending 0
All tests passed.
```

Reply:
`mvn clean install -Phadoop-3.2 -Phive-2.3 -Phadoop-cloud -Pmesos -Pyarn -Pkinesis-asl -Phive-thriftserver -Pspark-ganglia-lgpl -Pkubernetes -Phive -Pscala-2.13 -pl mllib-local`

all test passed.

```
Run completed in 2 seconds, 18 milliseconds.
Total number of tests run: 95
Suites: completed 9, aborted 0
Tests: succeeded 95, failed 0, canceled 0, ignored 0, pending 0
All tests passed.
```

Reply:
`mvn clean install -Phadoop-3.2 -Phive-2.3 -Phadoop-cloud -Pmesos -Pyarn -Pkinesis-asl -Phive-thriftserver -Pspark-ganglia-lgpl -Pkubernetes -Phive -Pscala-2.13 -pl sql/catalyst`

all test passed.

```
Run completed in 12 minutes, 27 seconds.
Total number of tests run: 4718
Suites: completed 261, aborted 0
Tests: succeeded 4718, failed 0, canceled 0, ignored 5, pending 0
All tests passed.
```

Reply:
mvn clean install -Phadoop-3.2 -Phive-2.3 -Phadoop-cloud -Pmesos -Pyarn -Pkinesis-asl -Phive-thriftserver -Pspark-ganglia-lgpl -Pkubernetes -Phive -Pscala-2.13 -pl sql/core
```
Run completed in 1 hour, 38 minutes, 11 seconds.
Total number of tests run: 9185
Suites: completed 406, aborted 0
Tests: succeeded 9185, failed 0, canceled 1, ignored 51, pending 0
All tests passed.
```

Reply:
some related modules that have not been verified today : `hive`, `hive-thriftserver` and `yarn`

Reply:
Do you want to check out the other modules first and do this all at once?

Reply:
Yes, I'll check these today

Reply:
@srowen All modules are verified